beacon-chain/db/filters/filter.go

```diff
@@ -59,13 +59,13 @@ func (q *QueryFilter) Filters() map[FilterType]interface{} {
 }
 
 // SetRoot --
-func (q *QueryFilter) SetRoot(val [32]byte) *QueryFilter {
+func (q *QueryFilter) SetRoot(val []byte) *QueryFilter {
```

Comment:
why not 32 bytes ? all roots are of that size

Reply:
easier to do the conversion in the function than doing it every call